opentelemetry-exporter-geneva/geneva-uploader/src/payload_encoder/mod.rs

```diff
@@ -0,0 +1,76 @@
+mod lz4_chunked_compression;
+
+#[cfg(test)]
+mod tests {
```

Comment:
code looks good, I'm just curious why you put the tests here and not in the `lz4_chunked_compression.rs` file?

Reply:
Thanks @gruebel for reviewing. You are correct, the tests logically belong to `lz4_chunked_compression.rs`. Have moved them there.